roles/openshift_efk/tasks/main.yaml

```diff
@@ -0,0 +1,179 @@
+---
+
+  - fail: msg="This role requires the following vars to be defined: master_url, kibana_hostname, es_cluster_size"
+    when: "kibana_hostname is not defined or
+          es_cluster_size is not defined or
+          master_url is not defined"
+
+  - name: "Checking for logging project"
+    command: oc get project logging
+    register: logging_project
+    failed_when: "'FAILED' in logging_project.stderr"
+    tags:
+      - cleanup
+
+  - name: "Create logging project"
+    command: oadm new-project logging
+    when: logging_project.rc != 0
+    tags:
+      - build
+  - name: "Changing projects"
+    command: oc project logging
```

Comment:
You will also want to add a role dependency on the openshift_common role, so that you can take advantage of the openshift.common.admin_binary and openshift.common.client_binary variables in case the oc/oadm clients get renamed in the future. See https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_persistent_volumes/tasks/main.yml#L19-L27